zeppelin-interpreter/pom.xml

```diff
@@ -42,7 +42,7 @@
     <maven.plugin.api.version>3.0</maven.plugin.api.version>
     <aether.version>1.12</aether.version>
     <maven.aeither.provider.version>3.0.3</maven.aeither.provider.version>
-    <wagon.version>1.0</wagon.version>
+    <wagon.version>2.7</wagon.version>
```

Comment:
is this a required change?

Reply:
Yes, without this CI was failing with 
```
18:26:12,285 ERROR org.apache.zeppelin.spark.DepInterpreter:253 - Exception in DepInterpreter while interpret 
org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.apache.commons:commons-csv:jar:1.1 in central (https://repo.maven.apache.org/maven2/)
	at org.sonatype.aether.impl.internal.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:375)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetchArtifactWithDep(SparkDependencyContext.java:171)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetch(SparkDependencyContext.java:121)
	at org.apache.zeppelin.spark.DepInterpreter.interpret(DepInterpreter.java:250)
	at org.apache.zeppelin.spark.DepInterpreterTest.testDefault(DepInterpreterTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```

With this do I need to update the licence?

Reply:
probably https://github.com/apache/zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE#L214